cypress/test/buttons/buttons.cy.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { mount } from 'cypress/react';
+import Button from '../../../components/buttons/Button';
+import IconGithub from '../../../components/icons/Github'
+describe('Button component', () => {
+  it('renders a button without href', () => {
+    const text = 'Click me';
+    const type = 'button';
+    const icon = <IconGithub />;
+    mount(
+      <Button text={text} type={type} icon={icon} />
+    );
+    cy.get('[data-testid="Button-main"]').should('have.text', text);
+    cy.get('[data-testid="Button-main"]').should('have.attr', 'type', type);
+    cy.get('[data-testid="Button-link"]').should('not.exist');
+  });
+
```

Comment:
Add a case for testing if icon is not present in Button component.

Reply:
done